src/components/Splash.vue

```diff
@@ -0,0 +1,154 @@
+<template>
+  <div class="flex flex-col justify-center" @wheel="preventScrolling">
+    <BasicIcon
+      name="splash"
+      iconClass="w-11/12 bp-500:w-9/12 md:w-6/12 lg:w-5/12 mt-24 sm:mt-16 place-self-center"
+    />
+
+    <div
+      class="bg-primary flex flex-col space-y-16 bp-360:space-y-14 bp-420:space-y-10 lg:space-y-12 items-center rounded-2xl py-12 bp-500:py-10 md:py-11 lg:py-12"
+    >
+      <!-- title -->
+      <p class="font-londrina text-white text-5xl">{{ displayTitle }}</p>
+
+      <!-- metadata -->
+      <div class="flex flex-col space-y-4 w-full items-center">
+        <div :class="metadataContainerClass">
+          <div :class="metadataCellClass" class="border-r-2">
+            <BasicIcon
+              name="question-mark-round"
+              :iconClass="metadataIconClass"
+            ></BasicIcon>
+            <div class="flex items-center">
+              <p :class="metadataTitleClass">{{ numQuestions }} questions</p>
+            </div>
+          </div>
+          <div :class="metadataCellClass">
+            <BasicIcon
+              name="student-in-class"
+              :iconClass="metadataIconClass"
+            ></BasicIcon>
+
+            <div class="flex items-center">
+              <p :class="metadataTitleClass">Class {{ classNumber }}</p>
+            </div>
+          </div>
+        </div>
+
+        <div :class="metadataContainerClass">
+          <div :class="metadataCellClass" class="border-r-2">
+            <BasicIcon name="math" :iconClass="metadataIconClass"></BasicIcon>
+            <div class="flex items-center">
+              <p :class="metadataTitleClass">{{ subject }}</p>
+            </div>
+          </div>
+          <div :class="metadataCellClass">
+            <BasicIcon
+              name="notepad"
+              :iconClass="metadataIconClass"
+            ></BasicIcon>
+            <div class="flex items-center">
+              <p :class="metadataTitleClass">{{ examType }}</p>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <!-- start button -->
+      <icon-button
+        :titleConfig="startButtonTextConfig"
+        buttonClass="bg-white hover:bg-gray-200 rounded-lg h-14 w-40  ring-primary px-2 border-b-outset border-primary"
+        class="rounded-2xl shadow-lg mt-4 place-self-center"
+        data-test="startQuiz"
+        @click="start"
+      ></icon-button>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import IconButton from "./UI/Buttons/IconButton.vue";
+import BasicIcon from "./UI/Icons/BasicIcon.vue";
+import { defineComponent, computed, reactive, toRefs } from "vue";
+export default defineComponent({
+  name: "Splash",
+  components: {
+    IconButton,
+    BasicIcon,
+  },
+  props: {
+    title: {
+      type: String,
+      required: true,
+    },
+    subject: {
+      type: String,
+      required: true,
+    },
+    examType: {
```

Comment:
Why `exam` and not `quiz`? `assessment` is the most general IMO since we'll use this same player for quizzes, tests, exams, and so on each with slightly different wording

Reply:
agreed - let's keep `quiz` as we are using that everywhere for now.